### PR TITLE
doc: translate zh/userguide/clustermanager/cluster-registration

### PR DESCRIPTION
**Pull request by B1F030**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
translate zh/userguide/clustermanager/cluster-registration

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/clustermanager/cluster-registration.md [hunk 1]
```diff
@@ -1,106 +1,102 @@
...
+在成员集群的 Kubernetes 控制平面中，我们同样需要成员集群的 `kubeconfig` 文件。紧接着，我们需要执行上述提供的 `karmadactl register` 命令的输出内容。
 
-```
+```bash
 karmadactl register 10.10.x.x:32443 --token t2jgtm.9nybj0526mjw1jbf --discovery-token-ca-cert-hash sha256:f5a5a43869bb44577dba582e794c3e3750f2050d62f1b1dc80fd3d6a371b6ed4
 ```
 
-```
+```shell
 # The example output is shown below
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/clustermanager/cluster-registration.md:100]:
```suggestion
# 输出的示例如下
```

**Comment by zhzhuang-zju** [comment]
/lgtm
cc @RainbowMango

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**